x/evm/keeper/keeper.go

```diff
@@ -391,3 +397,7 @@ func (k Keeper) AddTransientGasUsed(ctx sdk.Context, gasUsed uint64) (uint64, er
 	k.SetTransientGasUsed(ctx, result)
 	return result, nil
 }
+
+func (k Keeper) GetSdkCtx(handlerId uint64) *sdk.Context {
+	return k.sdkCtxs[handlerId]
```

Comment:
```suggestion
	return k.sdkCtxs[evmID]
```

@kenta-mori3322 I'm looking at your 2 PRs, and I propose to rename all `handlerID` to `evmID`. Does that make sense to you?

Reply:
Done.